.github/workflows/e2e-tests.yml

```diff
@@ -135,5 +135,7 @@ jobs:
         uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: e2e-screenshots
-          path: tests/e2e/screenshots
+          name: e2e-debugging-wp[${{ matrix.wp_version }}]-amp[${{ matrix.amp_version }}]-run[${{ github.run_attempt }}]
+          path: |
+            ~/.npm/_logs/
+            tests/e2e/screenshots/
```

Comment:
This change makes a few enhancements:

- includes npm logs in the artifact for inspection
- updates the zip name to be unique to each run + attempt to avoid conflicts/collisions as GHA will not do anything to automatically